react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
+You may have used a GraphQL client library that was at least partly view layer agnostic, but still able to work with React or other view layer solution in the form of applications like Apollo Client, Urql and Relay. The next chapter is to illustrate how to implement a GraphQL client library that works with React. The takeaway shouldn't be "build your own GraphQL client for your production ready applications", however, you should instead learn:
 
-* see how a GraphQL client library works under the hood
-* see how it isn't too difficult to implement it your way
-* see how you would be able to contribute to the GraphQL ecosystem
+* how a GraphQL client library works under the hood
+* how simple a GraphQL client library is to implement your way
+* how it contributes to the GraphQL ecosystem
 
-I believe there is great potential to contribute to the GraphQL ecosystem, because it is fairly new. It would be great to have a diverse set of tools out there and not only one ecosystem pushing the whole thing forward. So see it as your opportunity to contribute to it.
+There is contribute to the GraphQL ecosystem, because the tools surrounding it are still fairly new. A diverse set of tools would speed this along insgtead of the one ecosystem pushing its maturation forward. This is not only a useful addition for your web development skillset, it is alos an opportunity to contribute to the early stages of a software's growth.
 
-Before diving into the single parts of implementing your own GraphQL client for React, let's think about the essential parts to be able to consume a GraphQL API in your React application?
+Before diving into implementing your own GraphQL client for React, consider the essentials for consuming a GraphQL API in a React application:
 
-* First, there must be **the GraphQL client itself**. It can be any HTTP library or even the {{% a_blank "native fetch API" "https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" %}}, but it must be able to send HTTP methods with a payload across the wire. Note: Even though the GraphQL specification isn't opinionated about the transportation layer, the GitHub GraphQL API (which you are going to consume with your own GraphQL client) is using HTTP. Thus our GraphQL client has to be able to execute GraphQL operations by using HTTP methods.
+1. The **GraphQL client** must be used. It can be any HTTP library or even the {{% a_blank "native fetch API" "https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" %}}, but it must be able to send HTTP methods with a payload across the wire.  While the GraphQL specification isn't opinionated about the transportation layer, the GitHub GraphQL API you  consume with GraphQL client is using HTTP. Our GraphQL client must be able to execute GraphQL operations by using HTTP methods.
```

Comment:
First, instead of 1. <- I have never used these enumerations before, so I wouldn't want to start here without updating all other material.

Reply:
"The **GraphQL client** must be used." to "A **GraphQL client** must be used."

Reply:
"you  consume with GraphQL client is using HTTP" to "you would consume with a GraphQL client is using HTTP"

Reply:
"Our GraphQL client must be able to execute GraphQL operations by using HTTP methods." to "So in conclusion, our GraphQL client must be able to execute GraphQL operations by using HTTP methods."

Reply:
* I changed the enumerated list to a normal bulleted list.  Adding "First, second, third" to a listing is redundant, and poor grammar.  You already have the items separated, and the user will read them in order without the need to tell them which is first and which is third.
* Changes made, no notes.
* Saying "you would consume" is a more vague way of saying "you consume". If consuming a GraphQL API with a GraphQL client ALWAYS uses HTTP, then you can say it directly.
* Starting a sentence with "so" is incorrect.  Also, you don't ever need to say "in conclusion" for the last sentence of a paragraph in English.  It is ALWAYS assumed the last sentence is the conclusion for that paragraph.  It just adds extra words that throw off the flow of your writing.